server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0044_FixDownloadedChaptersWithoutPageCountII.kt

```diff
@@ -0,0 +1,23 @@
+package suwayomi.tachidesk.server.database.migration
+
+/*
+ * Copyright (C) Contributors to the Suwayomi project
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import de.neonew.exposed.migrations.helpers.SQLMigration
+
+@Suppress("ClassName", "unused")
+class M0044_FixDownloadedChaptersWithoutPageCountII : SQLMigration() {
+    override val sql: String =
+        """
+        UPDATE CHAPTER
+        SET IS_DOWNLOADED = FALSE
+        WHERE ID IN (
+            SELECT ID FROM CHAPTER
+            WHERE IS_DOWNLOADED = TRUE AND PAGE_COUNT <= 0
+        );
```

Comment:
Isn't this more optimized?
```suggestion
        UPDATE CHAPTER
        SET IS_DOWNLOADED = FALSE
        WHERE IS_DOWNLOADED = TRUE AND PAGE_COUNT <= 0;
```